lib/content-services/src/lib/node-comments/node-comments.component.spec.ts

```diff
@@ -0,0 +1,78 @@
+/*!
+ * @license
+ * Copyright © 2005-2025 Hyland Software, Inc. and its affiliates. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { Component } from '@angular/core';
+import { By } from '@angular/platform-browser';
+import { NodeCommentsComponent } from './node-comments.component';
+import { CommentsComponent, CommentModel, ADF_COMMENTS_SERVICE } from '@alfresco/adf-core';
+import { NodeCommentsService } from './services/node-comments.service';
+
+@Component({
```

Comment:
You don't need that mocked component. You should test NodeCommentsComponent directly. 
To test it directly you need to spyOn emit function from commentAdded field and then trigger commentAdded action from CommentsComponent. After that you can check if triggering that event caused calling emit on commentAdded from NodeCommentsComponent